### PR TITLE
chaos(dm): fix coredns problem

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve?

Issue Number: ref #5738 

### What is changed and how it works?

port #5735 

#### Tests 

 - https://github.com/pingcap/tiflow/actions/runs/2445689584

### Release note 

```release-note
 `None`.
```

### CODE REVIEW

**Review by amyangfei — APPROVED** [review]

File: .github/workflows/dm_chaos.yaml [hunk 1]
```diff
@@ -120,10 +98,11 @@ jobs:
...
           docker build -f $GITHUB_WORKSPACE/dm/chaos/manifests/Dockerfile -t dm:chaos $GITHUB_WORKSPACE/bin
           docker image list
+
       # Load DM docker image into KIND, see https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster
-      # - name: Load DM docker image into KIND
-      #   run: |
-      #     kind load docker-image dm:chaos --name dm-chaos
+      - name: Load DM docker image into KIND
+        run: |
+          kind load docker-image dm:chaos
```
Comment (amyangfei) [inline .github/workflows/dm_chaos.yaml:105]: default name for kind cluster is `chart-testing`, or we can configure with `cluster_name` when setup kind

**Comment by lance6716** [comment]
chaos is OK

https://github.com/pingcap/tiflow/actions/runs/2445689584

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**